### PR TITLE
Configuration of Material-UI

### PR DESCRIPTION
**Pull request by david8611**
This is hopefully the correct configuration of the Material-UI framework using react-native-paper.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,12 +8,15 @@
...
+    "@material-ui/core": "^4.11.2",
     "expo": "~39.0.2",
     "expo-status-bar": "~1.0.2",
     "react": "16.13.1",
     "react-dom": "16.13.1",
     "react-native": "https://github.com/expo/react-native/archive/sdk-39.0.4.tar.gz",
-    "react-native-web": "~0.13.12"
+    "react-native-paper": "^4.7.0",
+    "react-native-web": "~0.13.12",
+    "react-router-dom": "^5.2.0"
```
Comment (david8611) [inline package.json:19]: This could possibly be removed since we aren't yet using links to switch between pages.

**Comment by david8611** [comment]
I also don't know why the package-lock.json file was so drastically changed. Although, it seems to work since there are no compilation errors or execution errors with the exception of a font (Helvetica) error.

File: package.json (hunk 1 above)
Comment (david8611) [inline package.json:19]: This could possibly be removed since we aren't yet using links to switch between pages.

**Comment by jscarlsson** [comment]
My understanding is that package-lock.json is basically for ensuring we all have the exact same packages installed, so it should get updated when you add stuff. We also updated it (automatically) meaning there might be an annoying merge conflict, but basically the solution is to include everything if I understand correctly (I could help with that if it comes to that, but it's not like I have a great understanding of this either).

**Comment by david8611** [comment]
Yes, I will try to allow all (keep both) when fixing the conflicts. I think this is the only main (big) conflict if we compare each file. There might be some other conflicts in the regular package.json file as well but those I think will be easally solved by allowing both, since we're both adding eslint and react-native-paper (MaterialUI) modules/packages.

**Review by sanlil — APPROVED** [review]
Looks good to me! 🌈 

Don't forget to [update the theme](https://material-ui.com/customization/theming/) with the Ovulai brand colors as well

**Comment by david8611** [comment]
> Looks good to me! 🌈
> 
> Don't forget to [update the theme](https://material-ui.com/customization/theming/) with the Ovulai brand colors as well

Yes, this will be added in another PR.

**Pull request merged**